src/plugins/condition/ConditionManager.js

```diff
@@ -139,10 +146,17 @@ export default class ConditionManager extends EventEmitter {
         }
     }
 
+    getConditionDescription(condition) {
+        condition.updateDescription();
```

Comment:
getConditionDescription is calling updateDescription on condition, feels incorrect. May be rename something else?